### DISCUSSION TITLE
Change handyman to janitor for inclusivity

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#21939] [Feature requests]
janitor is gender neutral whil handyman is not

**Comment by harvito** [comment]
I'm for this. Maybe handyworker or one of these to keep the mood
A handyman (abbr. HNDMN), also known as a fixer, handyperson or handyworker, maintenance worker, repair worker, or repair technician...
\- [Wikipedia](https://en.wikipedia.org/wiki/Handyman)

Or honestly janitor since it's pretty accurate
